app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -26,7 +29,7 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
   return (
     <Formik
       initialValues={note}
-      validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
+      validationSchema={VALIDATION_SCHEMA}
       onSubmit={handleSubmit}
     >
```

Comment:
Pass in the `noValidate` prop. Otherwise both the default browser validations will kick in for the fields along with the Formik validations.

Reply:
@navaneethsdk I have made changes to the files under the `Pane` for localization and renaming of two constants, namely `VALIDATION_SCHEMA` and `INITIAL_FORM_VALUES`. The remaining changes to this `pane` are yet to be made as part of #4.